src/main/java/frc/team2412/robot/commands/launcher/FullTargetCommand.java

```diff
@@ -0,0 +1,93 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.interpolation.InterpolatingTreeMap;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.team2412.robot.Controls;
+import frc.team2412.robot.Robot;
+import frc.team2412.robot.subsystems.DrivebaseSubsystem;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+import frc.team2412.robot.util.LauncherDataLoader;
+import frc.team2412.robot.util.LauncherDataPoint;
+import java.nio.file.FileSystems;
+import java.util.function.BooleanSupplier;
+
+public class FullTargetCommand extends Command {
+
+	private static final InterpolatingTreeMap<Double, LauncherDataPoint> LAUNCHER_DATA =
+			LauncherDataLoader.fromCSV(
+					FileSystems.getDefault()
+							.getPath(Filesystem.getDeployDirectory().getPath(), "launcher_data.csv"));
+	private static final float YAW_TARGET_VIBRATION_TOLERANCE = 3; // degrees
+	private final Pose2d SPEAKER_POSE = new Pose2d(0.0, 5.55, Rotation2d.fromRotations(0));
```

Comment:
When will handling of different alliances be added?

Reply:
I dont think we need to add handling for this as the field is symmetrical along the center line this year

Reply:
What's the plan for the coordinate system when on blue alliance vs when on red alliance?

Reply:
nvm i was confused by how PathPlanner was handling alliance switching